src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/domain/enums/ChangeLevel.kt

```diff
@@ -1,6 +1,7 @@
 package uk.gov.justice.digital.hmpps.hmppstier.domain.enums
 
 enum class ChangeLevel {
+  ZERO,
```

Comment:
I had this and removed it previously, I guess its semantics, but is there a difference between a 0 and a null, and what do we want them to mean?